### PR TITLE
FIP0057: Update implementation & product considerations

### PR DESCRIPTION
**Pull request by Stebalien**
1. Include the results of the gas backtesting.
2. Link to the implementation.

### CODE REVIEW

File: FIPS/fip-0057.md [hunk 1]
```diff
@@ -473,11 +473,22 @@ As with all gas schedule changes, this FIP introduces changes to the operation c
 
 ## Product Considerations
 
-This FIP will likely increase gas fees in some cases and needs to be carefully benchmarked.
+Overall, due to optimizations in the built-in actors, gas usage for most storage provider messages is expected to _decrease_ by ~30% on average (40% for  `PublishStorageDeals`).
```
Comment (anorth) [inline FIPS/fip-0057.md:476]: Is this a statement about changes coming with FIP-0057, or orthogonal optimisation work that we did? I.e. is it about this FIP, or about network v18? If the latter, it probably doesn't belong here.

**Review by anorth — APPROVED** [review]

File: FIPS/fip-0057.md (hunk 1 above)
Comment (Stebalien) [reply FIPS/fip-0057.md:476]: > Is this a statement about changes coming with FIP-0057

This statement comes from:

1. Slightly lower per-block write fee.
2. Optimizations applied in the actors build system (wasn't officially a part of any FIP, although I can add it to this one, I guess).

I think it is important to note that any impact of this FIP is, for most use-cases, offset by other optimizations that will be shipping alongside it.

Comment (kaitlin-beegle) [reply FIPS/fip-0057.md:476]: I agree that we should note the relationship this FIP has to expected gas fee changes, but the specifics of a change as caused by a non-FIP bug fix, enhancement, etc., can be accounted for in the Core Devs repo. 

A bit of a follow-on, but how formalized is network monitoring and benchmarking after an upgrade? Does EngRes or some other entity within Protocol Labs actively monitor network conditions for X amount of time, or is the monitoring more passive?  E.g., individuals check Starboard or other network usage dashboards, wait to ensure systems are working properly, and respond whenever a red flag goes up from some component of the protocol? I'm not sure how to make my question more specific so please let me know if it's unclear.  

FIPs often make reference to 'monitoring network impact', but I rarely see follow-up information about network effects post-implementation. That's not to say it isn't happening, it's just not visible to me, and I'm curious.

Comment (jennijuju) [reply FIPS/fip-0057.md:476]: > but how formalized is network monitoring and benchmarking after an upgrade

Very good question @kaitlin-beegle. Yes - we(Lotus and Sentinel) develops  monitoring system for each upgrade, including regular network health metrics like block production and utilization, messages, and etc, and additional monitoring  according to the features/adjustment being added to the protocol from this upgrade. 
Core network deployment teams actively monitor the network for 12-24 hours(differs based on the size of each upgrade), and Lotus also continuously monitor the network health as the upgrade stabilizes over the week. We also develop alerting systems for things thats alarming for the network health.


All the information can be derived from the chain itself, and we welcome and encourage the community/ecosystem to develop toolings and monitor the network health along with PL engres. If this is something anyone would love to chat more about, feel free to reach me #fil-lotus-dev in [Filecoin slack](filecoin.io/slack).

**Review by jennijuju — APPROVED** [review]

**Pull request merged**